spec/routing/conferences_routing_spec.rb

```diff
@@ -10,19 +10,19 @@
       expect(get('/conferences/1')).to route_to('conferences#show', id: '1')
     end
 
-    it 'routes to #edit' do
+    xit 'routes to #edit' do
```

Comment:
What should I do with this routing tests? I can move them to an spec/routing/orga/conferences_routing_spec.  But do we need routing_specs in the regular controller as well?